aepp/flowservice.py

```diff
@@ -358,6 +363,19 @@ def getConnectionSpec(self, specId: str = None) -> dict:
         res: dict = self.connector.getData(self.endpoint + path)
         return res.get('items',[{}])[0]
 
+    def getConnectionSpecIdFromName(self, name: str = None) -> int:
+        """
+        Returns the connection spec ID corresponding to a connection spec name.
+        Arguments:
+            name : REQUIRED : The specification name of a connection
+        """
+        if name is None:
+            raise Exception("Require a name to be present")
```

Comment:
should we log these exceptions as well?

Reply:
I let you decide here.

Reply:
For now let's keep it as-is since the same pattern for raising exceptions is used throughout the repo, we can revisit separately.